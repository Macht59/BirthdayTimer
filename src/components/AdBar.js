import React from "react";
import { View, AppRegistry } from "react-native";
import {
  AdMobBanner,
  AdMobRewarded,
  AdMobInterstitial,
  PublisherBanner
} from "react-native-admob";

export default class AdBar extends React.Component {
  render() {
    return (
      <View>
        <AdMobBanner
          adSize="banner"
          adUnitID="ca-app-pub-7409231048157456/4292105638"
          ref={el => (this._basicExample = el)}
          style={{alignSelf: "center"}}
          onAdFailedToLoad={this.onAdFailedToLoad}
        />
      </View>
    );
  }
  onAdFailedToLoad(a){
    console.log("onAdFailedToLoad");
    console.log(JSON.stringify(a));
  }
}

AppRegistry.registerComponent("AdBar", () => AdBar);
