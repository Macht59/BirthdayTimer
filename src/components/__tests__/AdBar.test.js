import React from 'react';
import 'react-native'
import AdBar from '../AdBar';
import renderer from 'react-test-renderer';

test('renders without crashing', () => {
    const rendered = renderer.create(<AdBar />).toJSON();
    expect(rendered).toMatchSnapshot();
});