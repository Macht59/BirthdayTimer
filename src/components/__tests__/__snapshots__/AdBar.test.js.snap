// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnBar component renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "alignSelf": "center",
      }
    }
  >
    <RNAdMob
      adUnitID="ca-app-pub-7409231048157456/8013801446"
      bannerSize="banner"
      onDidFailToReceiveAdWithError={[Function]}
      onSizeChange={[Function]}
      style={Object {}}
    />
  </View>
</View>
`;
